### DISCUSSION TITLE
Representative band structure plotting script

### DISCUSSION THREAD

**Discussion by wladerer** [irreducible-representations/irrep#93] [Q&A] [answered]
Hi! Thank you for putting this all together. I was wondering if anyone had an example python script to plot the irreducible representations from the .json file over a VASP band structure calculation? 

I imagine I could run a line mode calculation connected by the maximal kpoints and then overlay that with a scatter plot of the various irreps from the json file?

Thank you!

**Comment by MIraola** [comment, marked as answer]
Hi Will Laderer,

currently, we don't distribute such scripts with the code. Indeed, I don't have any script for that. 

Nevertheless, it is not hard to write such a script. You can parse the `.json` output that IrRep generates. I also recommend having a look at the `-plotbands` argument. You can get more info about it in IrRep's help interface: `irrep --help`. 

Best,
Mikel I.

By the way, sorry for the delay. I just noticed that this section exists :S
